recipes/pycoalescence/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "pycoalescence" %}
+{% set version = "1.2.6rc42" %}
+{% set sha256 = "d69eeeed6837db8113271d05b9ca6fa95c89a55f27676a30fe12b6941918713b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  detect_binary_files_with_prefix: True
+  skip: True  # [win or not py3k]
```

Comment:
I'm skipping windows because I tested that locally in a linux machine

You have to do just the same thing in windows as I did for linux

Reply:
Looks great, thanks for your help. I'm testing on a few systems (think it might be possible without the build scripts), so will give it a try with this and then can ping you when I think it's working properly so as to not waste more of your time.

Reply:
Yes, sure! Please fell free to ping me  :)